transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/WsebTransportTest.java

```diff
@@ -79,12 +86,7 @@
 	private NioSocketAcceptor tcpAcceptor;
 	private HttpAcceptor httpAcceptor;
 	private WsebAcceptor wsebAcceptor;
-
-    @BeforeClass
-    public static void debugging() throws Exception {
-        PropertyConfigurator.configure("src/test/resources/log4j-trace.properties");
-    }
-
+	
```

Comment:
Yes I think we could use it. I actually created it by copying code from this test! But I'd rather hold off until we do the work of building a WsebAcceptorRule too and including all the wseb spec tests.
Once we do that we can actually remove this test.